Sources/Testing/ExitTests/ExitTest.Condition.swift

```diff
@@ -19,13 +18,29 @@ extension ExitTest {
   ///
   /// Values of this type are used to describe the conditions under which an
   /// exit test is expected to pass or fail by passing them to
-  /// ``expect(exitsWith:observing:_:sourceLocation:performing:)`` or
-  /// ``require(exitsWith:observing:_:sourceLocation:performing:)``.
+  /// ``expect(processExitsWith:observing:_:sourceLocation:performing:)`` or
+  /// ``require(processExitsWith:observing:_:sourceLocation:performing:)``.
+  ///
+  /// ## Topics
```

Comment:
Related to [this comment](https://github.com/swiftlang/swift-testing/pull/324/files#r2069450546), I noticed while reviewing the rendered docs that `init(_:)` is not included in this Topics list and it's getting implicitly added at the bottom in an "Initializers" section. I suppose that's not a problem, but wanted to point it out.